guides/en/dateTime.md

```diff
@@ -0,0 +1,43 @@
+# Date Time Cheatsheet
+## Formatting
+
+| Format | Notes                                                              |
+|--------|--------------------------------------------------------------------|
+| d      | Day of the month as digits; no leading zero for single-digit days. |
+| dd     | Day of the month as digits; leading zero for single-digit days.    |
+| EEE    | Day of the week as a three-letter abbreviation.                    |
+| EEEE   | Day of the week as its full name.                                  |
+| m      | Month as digits; no leading zero for single-digit months.          |
```

Comment:
So --  the `m` means minutes in `timeFormat`, but month in `dateTimeFormat` that should be clarified and also check to see if there are any other conflicting mask names between the functions.